pkg/models/commandvalue.go

```diff
@@ -27,7 +26,8 @@ const (
 	MaxBinaryBytes = 16777216
 	// DefaultFoloatEncoding indicates the representation of floating value of reading.
 	// It would be configurable in system level in the future
-	DefaultFloatEncoding = models.Base64Encoding
+	DefaultFloatEncoding = "Base64"
+	ENotation            = "eNotation"
```

Comment:
any specific reason to make this change ?

Reply:
We should totally remove this part.  There is no base64 encoding anymore

Reply:
Modified and upgrade dependency lib.